git/git.go

```diff
@@ -59,14 +59,14 @@ func LsRemote(remote, remoteRef string) (string, error) {
 		return "", errors.New("remote required")
 	}
 	if remoteRef == "" {
-		return simpleExec("git", "ls-remote", remote)
+		return subprocess.SimpleExec("git", "ls-remote", remote)
```

Comment:
My first pass was like that but then I read https://blog.golang.org/package-names and realised it was considered a 'bad package name' so I changed it :smile: